services/user.py

```diff
@@ -5,17 +5,25 @@
 def create_user(
         username: str,
         password: str,
-        email: str = "",
-        first_name: str = "",
-        last_name: str = ""
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None

```

Comment:
You can replace emal, first_name, last_name with **kwargs. Then you need to create some list/tuple with "email", "first_name", "last_name", and create your object 'User' by username and password. Then you need to unpack your kwargs and iterate one by one, checking if such key exist in your kwargs.items() and checking if value of this key is not "" or None. In this case after your check you can use setattr() to add this value to your 'User' object